src/api-error.js

```diff
@@ -15,13 +15,11 @@ class APIError extends Error {
     this.status = status;
   }
 
-  toString = () => {
-    return `\n${this.message}\nResponse:\n${JSON.stringify(
-      this.response,
-      null,
-      2,
-    )}`;
-  };
+  toString() {
+    return `${this.message}
+    Response:
+    ${JSON.stringify(this.response, null, 2)}`;
+  }
```

Comment:
FYI, this is going to format the error messages like this:

```
[Message]
    Response:
    { .. }
```

This is probably not what you want - you'll want to remove the leading spaces on the second and third lines.

Reply:
addressed in #78, granted it looks ugly now imo 😂